data/json/furniture_and_terrain/furniture-tools.json

```
@@ -1446,5 +1446,26 @@
       "sound_fail": "whump.",
       "items": [ { "item": "grip_hook", "count": [ 4, 4 ] }, { "item": "rope_30", "count": [ 1, 1 ] } ]
     }
+  },
+  {
+    "type": "furniture",
+    "id": "f_web_up",
+    "name": "web leading up",
+    "looks_like": "fd_web",
+    "description": "A long, thick strand of slightly sticky web.  You could climb it up.",
```

Awkward wording due to being similar to "you could climb up it".
```suggestion
    "description": "A long, thick strand of slightly sticky web.  You could use it to climb up.",
```